### ISSUE TITLE
[Housekeeping] Verify flyteadmin health check with auth & SSL

### ISSUE THREAD

**Issue by evalsocket** [flyteorg/flyte#1471]
**Describe the Issue**
Currently, we added a pre-check-in flytescheduler. Pre-check will check flyteadmin before starting the flytescheduler. 

Currently, we didn't test it with auth, Ideally, flyteadmin health check will not have the auth 

Related to https://github.com/flyteorg/flyteadmin/pull/254

**What if we do not do this?**
Short description of potential consequences

***Related component***
Either Specific / all

**Comment by evalsocket** [comment]
cc @pmahindrakar-oss

**Comment by evalsocket** [comment]
@pmahindrakar-oss i think we can close this

**Comment by pmahindrakar-oss** [comment]
Can we use this to introduce new secrets for scheduler to talk to admin.Currently we use propeller auth. SSL path is currently not required since scheduler talks to admin directly without ingress.we can recheck about union cloud requirements separately if they are diff but we can repurpose this just for introducing new client secrets .if it's already done then we can close it

**Comment by evalsocket** [comment]
cc: @EngHabu @katrogan

**Comment by katrogan** [comment]
what does

> flyteadmin health check will not have the auth

 mean?

**Comment by evalsocket** [comment]
@pmahindrakar-oss @katrogan Any update on this issue?

**Comment by katrogan** [comment]
@pmahindrakar-oss want to merge https://github.com/flyteorg/flyteadmin/pull/417?

**Comment by pmahindrakar-oss** [comment]
Awaiting review on this from @EngHabu https://github.com/flyteorg/flyte/pull/2431
